cli/lib/import/cubeMetadata.ts

```diff
@@ -37,8 +37,23 @@ function * extractExistingMetadata(dataset: DatasetExt, graph: Term, id: Term):
   }
 }
 
+const mirroredProperties: [NamedNode, NamedNode][] = [
+  [schema.name, dcterms.title],
+  [schema.description, dcterms.description],
+]
```

Comment:
i am kinda.. because this is deemed to get missed when a new property gets added, no? At least make a huge big comment in the shape to also update this one.

Reply:
Added that comment.